tracery/modifiers.py

```diff
@@ -67,3 +67,130 @@ def lowercase(text, *params):
     'uppercase': uppercase,
     'lowercase': lowercase
 }
+
+
+#----
+#inflect.py modifiers
+#Mappings based on https://github.com/pwdyson/inflect.py
+
+import inflect
+p = inflect.engine()
+
+
+def ordinal(text, *params):
+    return p.ordinal(text)
+
+def number_to_words(text, *params):
+    kwargs=dict(param.split('=') for param in params)
+    try:
+        resp = p.number_to_words(text, **kwargs)
+    except:
+        resp = "no recorded"
+    return resp
+    
+
+def plural(text, *params):
+    if params: return p.plural(text, params[0])
+    return p.plural(text)
+
+def plural_noun(text, *params):
+    if params: return p.plural_noun(text, params[0])
+    return p.plural_noun(text)
+
+def plural_verb(text, *params):
+    if params: return p.plural_verb(text, params[0])
+    return p.plural_verb(text)
+
+def plural_adj(text, *params):
+    if len(params): return p.plural_adj(text, params[0])
+    return p.plural_adj(text)
+    
+def singular_noun(text, *params):
+    return p.singular_noun(text)
+    
+def no(text, *params):
+    return p.no(text)
+    
+def num(text, *params):
+    return p.num(text)
+    
+def compare(text, *params):
+    return p.compare(text, params[0])
+    
+def compare_nouns(text, *params):
+    return p.compare_nouns(text,params[0])
+
+def compare_adjs(text, *params):
+    return p.compare_adjs(text,params[0])
+
+def a(text, *params):
+    return p.a(text)
+
+def an(text, *params):
+    return p.an(text)
+
+def present_participle(text, *params):
+    return p.present_participle(text)
+
```

Comment:
a general comment on the following functions: `classical`, `gender`, `defnoun`, etc. are used by `inflect` to change the current state of the `inflect.engine` object. I'm not sure if this stateful model makes sense in the context of Tracery, since the way that the grammar is expanded may lead to these state-modifying functions being called in an unexpected order, leading to confusing behavior. can you give a quick example that demonstrates the utility of these functions? also (importantly), the `inflect.engine` object here is declared as a global variable, which means that the state of the object would be *maintained* between different calls to Tracery's `flatten`, which is definitely undesirable (each `flatten` should start with a fresh state). I'm not sure how to solve this problem. ideas?